formwidgets/mltextarea/partials/_mltextarea.htm

```diff
@@ -15,6 +15,7 @@
             placeholder="<?= e(trans($field->placeholder)) ?>"
             class="form-control field-textarea size-<?= $field->size ?>"
             autocomplete="off"
+            onchange="updateTextareaLayout(this)"
```

Comment:
```suggestion
            oninput="updateTextareaLayout(this)"
```

Reply:
@mjauvin The reason why I used onchange instead of oninput is because onchange is triggered far less:
![image](https://github.com/wintercms/wn-translate-plugin/assets/89913092/280d46e1-b21d-4ead-b769-249b7c66256b)
![image](https://github.com/wintercms/wn-translate-plugin/assets/89913092/5f765cc2-ef91-4634-8ff2-abb75623055f)

The translation button is hidden when the element is focussed, so it only needs to update once and thats when you unfocus the element.
For me it doesn't matter wether its change or input but change looks more performant to me but I guess its just minor

Reply:
The button is not hidden if the TailwindUi plugin is not used.

Reply:
This should be the same with or without that plugin...

Reply:
Z-index 10 on focus
![image](https://github.com/wintercms/wn-translate-plugin/assets/89913092/c3d23261-6339-4367-a19f-6392153995e0)

Reply:
But oninput is the safer choice 👍